client/src/js/analyses/components/Create/SelectedSamples.js

```diff
@@ -2,11 +2,10 @@ import React from "react";
 import styled from "styled-components";
 import { map } from "lodash-es";
 
-import { ListGroup } from "react-bootstrap";
-import { ListGroupItem } from "../../../base";
+import { ListGroupItem, BoxGroup, BoxGroupSection } from "../../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ListGroupItem' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4903821)